fullstop-apis/fullstop-violation-api/src/main/java/org/zalando/stups/fullstop/violation/ViolationType.java

```diff
@@ -51,4 +51,6 @@
 
     public static final String ARTIFACT_BUILT_FROM_DIRTY_REPOSITORY = "ARTIFACT_BUILT_FROM_DIRTY_REPOSITORY";
 
+    public static final String LAMBDA_FUNCTION_STARTED_FROM_UNTRUSTED_LOCATION = "LAMBDA_FUNCTION_STARTED_FROM_UNTRUSTED_LOCATION";
```

Comment:
maybe something like `EXECUTED` or `INVOKED` instead of `STARTED` to be more clear?

Reply:
creation and execution are two different operations for Lambda. Fullstop only gets notifications about creation/update, but not about invocations.